### ISSUE TITLE
Truncated passwords?

### ISSUE THREAD

**Issue by usernamenumber** [RusticiSoftware/TinCanPython#86]
Hopefully I'm just missing something obvious here, but I'm seeing some weird behavior when I try to use this library to submit a test record to my instance of learning locker. Other xAPI clients can submit, so I don't think it's a problem with the LRS. 

Here's a pretty-printed excerpt from the reult.data JSON blob I get back after attempting to submit a score (with auth credentials obfuscated, of course):
```
{u'code': 401,
 u'error': True,
 u'message': u'Unauthorized request.',
 u'success': False,
 u'trace': [{u'args': [u'6e5exxxxxxxxxxxxxxxxxxxxxxxxxx54bf',
                       u'74337d9e58xxxxxx'],
```

Notice that the password appears to be truncated. Is this just a side-effect of how the data is being displayed, or could tincan.py be munging the password badly for some reason? 

If anyone has tips on how to troubleshoot this and get it working, I would greatly appreciate it!

**Comment by jpablo128** [comment]
Hi. What kind of weird behavior? Take a look at my pull request, looks like your problem might be related to what I found.

**Comment by ajblum** [comment]
I had a similar issue trying to pass long username/passwords.  

I reported my issue with a workaround here: https://github.com/RusticiSoftware/TinCanPython/issues/85

**Comment by brianjmiller** [comment]
Should be fixed in 0.0.5 based on #88.
